### PR TITLE
feat: add vertifybamid2

### PR DESCRIPTION
**Pull request by MagdalenaZZ**
### This PR:

(If this is a release PR, no need to add following. Leave this part empty)
(Use the following lines to create a PR text body. Make sure to remove all non-relevant one after you're done)
(Repeat each field as many times as necessary)

Added: for new features.
Changed: for changes in existing functionality.
Deprecated: for soon-to-be removed features.
Removed: for now removed features.
Fixed: for any bug fixes.
Security: in case of vulnerabilities.

### Review and tests:
- [ ] Tests pass
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Code review
- [ ] `CHANGELOG.md` is updated
- [ ] New code is executed and covered by tests, and test approve
- [ ] Module compatibility section in `README.md` is up to date

### CODE REVIEW

**Review by elleira — APPROVED** [review]

File: docs/softwares.md [hunk 1]
```diff
@@ -297,4 +298,29 @@ Collects multiple statistics st once that can be used by MultiQC.
 
 #RESOURCESSCHEMA__picard_collect_multiple_metrics#
 
----
\ No newline at end of file
+
+
+## [verifybamid2](https://github.com/Griffan/VerifyBamID)
+Introduction to verifybamid2
```
Comment (jonca79) [inline docs/softwares.md:304]: Write short description here

File: .tests/integration/verifybamid2/1000g.phase3.10k.b38.exome.vcf.gz.dat.UD
Comment (jonca79) [inline .tests/integration/verifybamid2/1000g.phase3.10k.b38.exome.vcf.gz.dat.UD:1]: Remove these test files?

**Review by jonca79 — CHANGES_REQUESTED** [review]

File: docs/softwares.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (MagdalenaZZ) [reply docs/softwares.md]: Line removed, well spotted :-D)

File: docs/softwares.md (hunk 1 above)
Comment (MagdalenaZZ) [reply docs/softwares.md]: Added software description.

File: workflow/schemas/rules.schema.yaml [hunk 3]
```diff
@@ -73,7 +74,8 @@ properties:
             type: string
             description: Pileup file with allele information for selected sites
 
-  mosdepth:
+
+ mosdepth:
```
[the remark this answers is not in the document]
Comment (MagdalenaZZ) [reply workflow/schemas/rules.schema.yaml]: Indentation fixed

**Review by MagdalenaZZ — COMMENTED** [review]
Thanks @jonca79 for the review - I've tried to address all the points you mentioned.

**Pull request merged**